components/shared/styles.ts

```diff
@@ -89,8 +89,8 @@ export const TextBodyMicroBold = styled.p`
 // -- Buttons -- //
 
 export const PrimaryButton = styled.button<{
-  mod?: 'fillLight' | 'fillDark' | 'ghost' | false
```

Comment:
Was this a linting error? .. If so, it seems weird that we didn't catch it sooner

Reply:
The formatter caught and changed quite a few little things. Agreed that it is strange there were so many places I did not change but Prettier made adjustments.